mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.tmpl

```diff
@@ -918,6 +918,28 @@ is set to true. Defaults to ZONAL.`,
 				},
 				Description: `The replicas of the instance.`,
 			},
+			"replication_cluster": {
+				Type:     schema.TypeList,
+				Computed: true,
```

Comment:
Currently, there's no default value of replication_cluster (replication_cluster with all empty fields will be the default, but we can probably ignore it?), but I can't guarantee that remains true in future, because cloud SQL backend engs are constantly adding various fields everywhere.

If you think we don't need to take care of future issue, then I can remove "computed" now.

Reply:
We don't need to take care of future issue. One downside of `Computed: true` is that it prevents the removal of the field from the API request. Even the field is removed from the Terraform configuration, its previous value is used in the API request.